lib/zendesk_apps_support/package.rb

```diff
@@ -183,13 +183,10 @@ def compiled_templates(app_id, asset_url_prefix)
       end
     end
 
-    def market_translations!(locale)
-      result = translations[locale].fetch('app', {})
-      result.delete('name')
-      result.delete('description')
-      result.delete('long_description')
-      result.delete('installation_instructions')
-      result
+    def translations_for(locale)
+      trans = translations
```

Comment:
So what if I'm developing an app, running it with zat, and I'm changing my translations?